mypy/types.py

```diff
@@ -1376,8 +1376,21 @@ def deserialize(cls, data: JsonDict) -> Type:
 
 
 class ForwardRef(Type):
-    """Class to wrap forward references to other types."""
-    link = None  # type: Type # the wrapped type
+    """Class to wrap forward references to other types.
+
+    This is used when a forward reference to an (unanalyzed) synthetic type is found,
+    for example:
+
+        x: A
+        A = TypedDict('A', {'x': int})
+
+    To avoid false positives and crashes in such situations, we first wrap the second
```

Comment:
Should this be 'the first occurrence ...'?

Reply:
Yes, will fix this docstring.